### PR TITLE
Added info to build pdf locally

### LINKED ISSUE #556: Include instructions on how to build PDF
Currently, only those who know to look in `.circleci/config.yml` can figure out how to build the PDF (#537) . We should include instructions on how to do that.

High-level steps are 
1. Install Tex: https://www.latex-project.org/get/#tex-distributions
1. Run the various build commands to go from a clean checkout to the PDF

### PR DESCRIPTION
**Pull request by ankita240796**
closes #556 

#### What is included in this PR?
Adds documentation on building PDF locally.

### CODE REVIEW

File: shared-src/docs-developer-guide.rst [hunk 1]
```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
...
+
+The docs can be downloaded as PDF for the ease of users.
+
+To build the pdf locally, follow these steps:
+
+.. note::
+
+  All steps below explain the build process for odk1-docs. To build pdf for odk2-docs in a similar way, you can replace **1** with **2** in the above commands.
+
+1. Install XeLatex
```
Comment (yanokwa) [inline shared-src/docs-developer-guide.rst]: This only works for Ubuntu which might not be common. 
* Macs, https://www.tug.org/mactex
* Windows, https://miktex.org/

File: shared-src/docs-developer-guide.rst [hunk 2]
```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
 specify the check name in the check list
 in the function :py:func:`exclude_checks`
 in the file :file:`style-test.py`.
+
+.. _docs-pdf:
```
Comment (yanokwa) [inline shared-src/docs-developer-guide.rst]: This feels like it'd work better in docs-tech-guide right below Style-test your work? What do you think?

File: shared-src/docs-developer-guide.rst [hunk 3]
```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
...
+
+  .. note::
+
+    A docker image for XeLatex can be used as well instead of installing it on your system. Some the useful links for the same:
+
+      - https://github.com/moss-it/docker-xelatex
+      - https://github.com/thomasWeise/docker-texlive
+      - https://github.com/thomasWeise/docker-texlive
+
+2. Build latex for ODK docs. This can be done in following steps:
```
[the remark this answers is not in the document]
Comment (ankita240796) [reply shared-src/docs-developer-guide.rst]: Done!

File: shared-src/docs-developer-guide.rst [hunk 4]
```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
...
+  .. code-block:: console
+
+    $ sudo apt-get install texlive-xetex
+
+  .. note::
+
+    A docker image for XeLatex can be used as well instead of installing it on your system. Some the useful links for the same:
+
+      - https://github.com/moss-it/docker-xelatex
+      - https://github.com/thomasWeise/docker-texlive
```
[the remark this answers is not in the document]
Comment (ankita240796) [reply shared-src/docs-developer-guide.rst]: Oh, my bad, Fixed this!

File: shared-src/docs-developer-guide.rst (hunk 1 above)
Comment (ankita240796) [reply shared-src/docs-developer-guide.rst]: Added reference to this!

File: shared-src/docs-developer-guide.rst (hunk 2 above)
Comment (ankita240796) [reply shared-src/docs-developer-guide.rst]: Yes, agreed this should be a part of workflow. Made the changes!

File: shared-src/docs-developer-guide.rst [hunk 5]
```diff
@@ -340,3 +340,87 @@ To exclude an built-in proselint check,
...
+
+      $ make odk1-latex
+
+3. Resize all the images in the build directory to ensure proper rendering of images in pdf.
+
+  .. code-block:: console
+
+    $ python util/resize.py odk1-build
+
+4. Generate pdf for the produced tex file
```
[the remark this answers is not in the document]
Comment (ankita240796) [reply shared-src/docs-developer-guide.rst]: Done in #788 :smile:

**Pull request merged**